public/js/request.view.js

```diff
@@ -160,6 +160,8 @@ module.exports = BaseMapperView.extend({
     }else{
       mappedValues.body = null;
     }
+    mappedValues.url = mappedValues.url && mappedValues.url.trim();
+    mappedValues.mountPath = mappedValues.url && mappedValues.url.trim();
```

Comment:
Should not this rather be `mappedValues.mountPath = mappedValues.mountPath && mappedValues.mountPath.trim();` ?